tests/test.py

```diff
@@ -381,11 +381,11 @@ def test_line():
 
     out = h3.h3_line(h1, h2)
 
-    expected = {
+    expected = [
         '8928308280fffff',
         '89283082873ffff',
         '8928308287bffff'
-    }
+    ]
```

Comment:
@nrabinowitz Should the output of `h3_line` be considered an **ordered** list or an **unordered** set?

Reply:
The guarantees for this function are at https://github.com/uber/h3/blob/master/src/h3lib/lib/localij.c#L591 The output is ordered because "... every index in the line will be a neighbor of the preceding index."

Reply:
Got it. So there's no need to check for and remove zeros in the output?

Reply:
I don't think the output from the `h3Line` function can contain zeros.

Reply:
Thinking about this now, I wonder if `h3Path` would be a better name, to indicate to the user that this returns a path between two nodes in the cell adjacency graph.

Maybe something to consider when we look at naming in 4.0?

Reply:
Resolved this with https://github.com/uber/h3-py/pull/79/commits/32fe6a0ff34a1513f8fb038b2b1fb1f4d4ded908